src/main/java/com/softserve/teamproject/entity/Student.java

```diff
@@ -36,6 +36,22 @@
   @Column(name = "image")
   private byte[] image;
 
+  @Column(name = "image_name")
+  @Pattern(regexp = ".+(\\.jpg|\\.jpeg|\\.tiff|\\.png)",
+      message = "Sorry, you should use only allowed file types:"
```

Comment:
Compilator says, here I can use only String constant for message, not message from property. Have you idea how to put messages.property in this place? I see in entities it doesn't use now.